packages/core/src/external/carequality/ihe-gateway-v2/inbound/shared.ts

```diff
@@ -28,7 +28,12 @@ const isPurposeOfUseObject = (value: AttributeValue): value is { PurposeOfUse: C
 };
 
 export function convertSamlHeaderToAttributes(header: SamlHeader): SamlAttributes {
-  const attributes = header.Security.Assertion.AttributeStatement.Attribute;
+  const attributes = Array.isArray(header.Security.Assertion.AttributeStatement)
+    ? header.Security.Assertion.AttributeStatement?.[0]?.Attribute
```

Comment:
0 indexing here ok. AttributeStatement shouldnt be an array but some implementers send it to us as an array with one element.

Reply:
nit: use `toArray` and then get the first el